### ISSUE TITLE
[Bug] Create Sample Video checkbox in Add (import) page doesn't work

### ISSUE THREAD

**Issue by sjclayton** [curtwagner1984/YAPO#17]
Tried to import a new folder and generate the sample videos on import, and it didn't even attempt to do it, let alone throw an error of any sort... Even with the 'Create Sample Video?' check box checked it gave this when it completed.... saying the createSampleVideo box wasn't checked even though it was...

[25/Aug/2016 07:47:08] "GET /add-items/?actorsToAdd=&createSampleVideo=false&folderToAddPath=%2Fmedia%2Fshaun%2FArchive%2FPorn%2FAlaina+Dawson HTTP/1.1" 200

**Comment by curtwagner1984** [comment]
This should work now.

**Comment by sjclayton** [comment]
@curtwagner1984 Doesn't work on my end... 

```
Internal Server Error: /add-items/
Traceback (most recent call last):
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/shaun/YAPOenv/lib/python3.4/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/shaun/YAPOenv/YAPO/videos/views.py", line 566, in get
    videos.addScenes.get_files(folder_to_add_path, True)
  File "/home/shaun/YAPOenv/YAPO/videos/addScenes.py", line 46, in get_files
    create_scene(file_path, make_video_sample)
  File "/home/shaun/YAPOenv/YAPO/videos/addScenes.py", line 109, in create_scene
    make_sample_video(current_scene)
TypeError: 'bool' object is not callable
[25/Aug/2016 21:44:28] "GET /add-items/?actorsToAdd=&createSampleVideo=true&folderToAddPath=%2Fmedia%2Fshaun%2FArchive%2FPorn%2FAlessa+Savage HTTP/1.1" 500 95691

```

**Comment by sjclayton** [comment]
**UPDATE:** It's not a caching issue.... I just re-pulled it after going back to the previous commit, so I could play around with it some more.... 

It's acting so weird, it throws the above mentioned error at random times.... also I have found that checking the box sometimes works and other times it doesn't.... Also it will sometimes try to generate sample video even with out the box being checked, with the exact same randomly working and randomly erroring out result.

**Comment by curtwagner1984** [comment]
Yeah. It's because I'm an Idiot. There are two parts of this function: 
*  if the scene is already in the database and is missing a sample video
*  if the scene is not in the database yet.

Because the code for creating the sample is the same function for both of them, I only tested it with scenes that were already in the database and assumed it will work for the other.

However now I noticed that in the code for when the scene is not in the database, I mistakenly called a boolean `make_sample_video(current_scene)` instead of calling the actual function `create_sample_video(current_scene).`

Should work now...(really.)
